src/Integration.UnitTests/Binding/SolutionBindingOperation.WriterTests.cs

```diff
@@ -1,102 +0,0 @@
-//-----------------------------------------------------------------------
-// <copyright file="SolutionBindingOperation.WriterTests.cs" company="SonarSource SA and Microsoft Corporation">
```

Comment:
SolutionBindingOperation.Writer was deleted. The logic was split across the new service and the operation. The tests followed similar split.